GLAA.Common/GLAA.Common.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I like breaking this down, I toyed with putting the scheduler code in with the services but preferred it separate and would echo that here